### PR TITLE
Return matched name

### PR DESCRIPTION
**Pull request by sybrenjansen**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -47,36 +47,75 @@ Geo
...
-
-Each of the above normalization functions will return the match score as last entry in the tuple. These scores are
-always between 0.0 and 1.0, where 1.0 is a perfect match. If a known mapping exists, like ``Deutschland`` to
-``Germany``, then the match score will be 1.0.
+    """
+    Countries
+    """
+
+    normalize_country('Peoples rep. of China')
+    # [Match(canonical_name='China', matched_name='Peoples Republic of China', score=1.0)]
```
Comment (santhoshrajamanickam) [inline README.rst]: isn't `Match` replaced by `CountryMatch` and `LocationMatch` ? if so can you fix it in the README.md

Comment (sybrenjansen) [reply README.rst]: I already mentioned it in the README a bit up: 
```
In the output the ``NormalizedCountryMatch`` and ``NormalizedLocationMatch`` object names
have been replaced with ``Match`` to make the output more readable.
```

Comment (santhoshrajamanickam) [reply README.rst]: ah i missed that ... cool ...

**Review by santhoshrajamanickam — COMMENTED** [review]
Looks good .. after you make the changes to the `*Match` types ...i will approve it

**Review by santhoshrajamanickam — APPROVED** [review]
New change is more clear ...looks good!

**Pull request merged**